tests/run/test_coroutines_pep492.pyx

```diff
@@ -754,7 +754,8 @@ class AsyncBadSyntaxTest(unittest.TestCase):
             async def g(): pass
             await z
         await = 1
-        #self.assertTrue(inspect.iscoroutinefunction(f))
+        if sys.version_info > (3,10,5):
+            self.assertTrue(inspect.iscoroutinefunction(f))
 
 
 class TokenizerRegrTest(unittest.TestCase):
```

Comment:
it looks like `ns['foo']` isn't passing the duck-type check in inspect? I'm trying to debug this, how do I run just this one test case on my local machine?

Reply:
`python runtests.py test_coroutines_pep492` will run the file.

I don't think you can easily just run one testcase from the file though. I'd just delete the code you don't care about while you test it.

Reply:
(and if you're using the debugger to debug it then `--no-capture` will make things easier)

Reply:
You can select a test with `-k <regex>` and also use `--debug` to enable all options that help in debugging.

Reply:
woops, it's obvious why, the `inspect.is*function` changes are in 3.10.6+